tsup.config.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
+
+const tsup = require("tsup");
+const { umdWrapper } = require("esbuild-plugin-umd-wrapper");
+
+module.exports = tsup.defineConfig([
+  {
+    entry: {
+      "peggy.min": "./lib/peg.js",
+    },
+    format: ["umd"],
+    dts: true,
+    platform: "browser",
+    minify: true,
+    outDir: "browser",
+    target: "es5",
```

Comment:
This is negotiable.  Left in for now to generate a bunch of polyfills into the output.